doc/book/quick-start.md

```diff
@@ -49,7 +49,7 @@ return [
 <!-- ... -->
 
 <body>
-    <?= $this->navigation('default')->menu() ?>
+    <?= $this->navigation('navigation')->menu() ?>
```

Comment:
I think, this is the wrong way:

> Call the navigation helper with a parameter "navigation"?! Really strange!

[See my comment on the bug report.](https://github.com/zendframework/zend-navigation/issues/43#issuecomment-249105716)